content/contributing/helping-out/writing-docs.md

```diff
@@ -0,0 +1,237 @@
+The code in the book is compiled and tested to make sure that the examples work for readers.
+{% end %}
+
+{% callout() %}
+[See the `learning-code-examples` README.md for more info.](https://github.com/bevyengine/bevy-website/blob/main/learning-code-examples/README.md)
+{% end %}
+
+## Learning material structure
+
+As you probably noticed, our introductory learning material is split into two main sections:
+
+1. **Bevy Quick Start:** "Get started making your first game now!"
+2. **Bevy Book:** "Understand how Bevy works, and how you can use it"
+
+This is intended to cater to two different types of learners, without compromising the experience for either:
+
+* **Example-first:** These users want to dive right in, see everything in action and get a working game as quickly as possible.
+These users often have an idea in their mind that they want to start prototyping as quickly as possible.
+* **Definition-first:** These users want to carefully build up a mental model of Bevy, thoroughly understanding each new concept before moving on.
+These users tend to be driven by curiosity, or are aiming to carefully develop a new skill.
+
+Crucially, these paths are independent of the experience levels of the learner!
+Bevy intentionally aims to be inclusive of both complete beginners who have never programmed before, and professionals coming from other engines.
+
+| | **Beginner** | **Professional** |
+|-|-|-|
+| **Example-first** | Enthusiastic, wants to create a new version of the game they love. | Exploratory, wants to dive in and see how Bevy holds up in practice. |
+| **Definition-first** | Curious, wants to understand how making games works. | Critical, wants to understand Bevy's unique design choices. |
+
+Each of these requires their own complementary learning paths that branch as soon as they get to the [Learn page](@/learn/_index.md) to ensure that the first experience that they have with Bevy matches what they need.
+
+Once users have completed the introductory learning materials in their path of choice, they can begin creating their own games or move on to our advanced examples to see how everything comes together in a realistic way.
+
+### Bevy Quick Start: the example-first path
+
+Users following the example-first path will tend to take the following route:
+
+1. Encounter the Bevy homepage due to social media or word of mouth.
+2. Navigate to the Learn page.
+3. Choose one of the most relevant **quick start games**.
+4. Complete that tutorial.
+5. Dive into making the game they have in mind, accessing the following resources as needed when they encounter road-blocks:
+   1. Official Examples.
+   2. The Bevy book.
+   3. Community tutorials and template games.
+   4. Various community support forums.
+   5. Streams, YouTube channels and blogs.
```

Comment:
```suggestion
   5. Streams, videos, and blogs.
```
Oxford comma, and make the reference to videos generic since YouTube isn't the only video platform, nor did it fit in with the other two generic references.